### DISCUSSION TITLE
How to calculate the heat flux of a surface

### DISCUSSION THREAD

**Discussion by suqingji** [idaholab/moose#22065] [Q&A Modules: General] [answered]
Hi, 

I have finished the simulation of a conjugate heat transfer problem on complex geometry by the FEM NS module. Now I want to calculate the heat flux on the solid and fluid coupled surface. What should I use to calculate it? In Postprocessors I have not found a suitable class to finish it.

Thanks

**Comment by GiudGiud** [comment, marked as answer]
Hello

What object are you using for the heat transfer between the fluid and the solid?
The postprocessor will have to use the same discretization / formulation as that kernel/interface kernel

Guillaume
